deps/3rdparty/libdispatch/dispatch/dispatch.h

```diff
@@ -18,6 +18,8 @@
  * @APPLE_APACHE_LICENSE_HEADER_END@
  */
 
+#pragma once
```

Comment:
this pull request should be kept clean of 3rdparty file changes not necessary to build with VS2017.3